### ISSUE TITLE
onnx issues for ModuleList

### ISSUE THREAD

**Issue by bestlin** [pytorch/pytorch#3887]
I am wondering current onnx only supports Sequential operation but can not support ModuleList operation? How can I make the onnx support ModuleList?

**Comment by apaszke** [comment]
ONNX supports all kinds of containers. No need to change anything.

**Comment by cwhuang888** [comment]
Actually I have encountered the same conversion issue when handling a nested PyTorch model with ModuleList.
Following is a simple snippet of a real case.
Network: Network(
  (stem): Sequential(
    (0): Conv2d(3, 48, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
    (1): BatchNorm2d(48, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  )
  (cells): ModuleList(
    (0): Cell(
      (preprocess0): ReLUConvBN(
        (op): Sequential(
          (0): ReLU()
          (1): Conv2d(48, 16, kernel_size=(1, 1), stride=(1, 1), bias=False)
          (2): BatchNorm2d(16, eps=1e-05, momentum=0.1, affine=False, 
           track_running_stats=True)
        )
      ))))
In Python side, the graph node names are correctly obtained via the dictionary keys
'stem.0', 
'stem.1', 
'cells.0.preprocess0.op.0'
'cells.0.preprocess0.op.1', 
'cells.0.preprocess0.op.2',

However, graph nodes obtained from trace returned from torch.jit.get_trace_graph(model, tuple(args)) gives: 
Network/Sequential[stem]/Conv2d[0]
Network/Sequential[stem]/BatchNorm2d[1]
Network/Cell/ReLUConvBN[preprocess0]/Sequential[op]/ReLU[0]
Network/Cell/ReLUConvBN[preprocess0]/Sequential[op]/Conv2d[1]
Network/Cell/ReLUConvBN[preprocess0]/Sequential[op]/BatchNorm2d[2]

Where the last three should be
Network/**ModuleList[cells]/Cell[0]**/ReLUConvBN[preprocess0]/Sequential[op]/ReLU[0]
Network/**ModuleList[cells]/Cell[0]**/ReLUConvBN[preprocess0]/Sequential[op]/Conv2d[1]
Network/**ModuleList[cells]/Cell[0]**/ReLUConvBN[preprocess0]/Sequential[op]/BatchNorm2d[2]
